llarp/router/router.cpp

```diff
@@ -874,6 +874,60 @@ namespace llarp
     m_LastStatsReport = now;
   }
 
+  std::string
+  Router::status_line()
+  {
+    std::string status;
+#if defined(WITH_SYSTEMD)
+    auto out = std::back_inserter(status);
+    fmt::format_to(out, "WATCHDOG=1\nSTATUS=v{}", llarp::VERSION_STR);
```

Comment:
The return value here should be starting from "v", but not including the `WATCHDOG=1\nSTATUS=` prefix (which should be the responsibility of sd_service_manager to add).

Reply:
done